src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -5952,14 +5952,22 @@ private static string GetNamespaceToRemove(CompletionContext context, TypeComple
         internal static List<CompletionResult> CompleteHelpTopics(CompletionContext context)
         {
             var results = new List<CompletionResult>();
-            var dirPath = Utils.GetApplicationBase(Utils.DefaultPowerShellShellID) + "\\" + CultureInfo.CurrentCulture.Name;
+            var dirPath = Utils.GetApplicationBase(Utils.DefaultPowerShellShellID) + Path.DirectorySeparatorChar + CultureInfo.CurrentCulture.Name;
             var wordToComplete = context.WordToComplete + "*";
             var topicPattern = WildcardPattern.Get("about_*.help.txt", WildcardOptions.IgnoreCase);
-            string[] files = null;
+            ArrayList files = new ArrayList();
```

Comment:
Could you please use `List<string>` here? We prefer to strongly typed generic collection whenever possible, it will avoid a cast from `object` to `string` in the `foreach (string file in files)` loop later.